src/stale_deps.jl

```diff
@@ -5,7 +5,13 @@ Test that `package` loads all dependencies listed in `Project.toml`.
 Note that this does not imply that `package` loads the dependencies
 directly, this can be achieved via transitivity as well.
 
-!!! note "Known bug"
+!!! note "Weak dependencies and extensions"
+
+    Due to the automatic loading of package extensions once all of
+    their dependencies are loaded, a package extension can, by design,
+    not have any stale dependencies.
```

Comment:
```suggestion
    their trigger dependencies are loaded, Aqua.jl can, by design of julia,
    not check if a package extension indeed loads all of its trigger dependencies using `import` or `using`. 
```

@gdalle I understand your point now better. What do you think about this?

Reply:
yeah I think this is better!